cri/v1alpha1/cri.go

```diff
@@ -697,12 +725,55 @@ func (c *CriManager) ContainerStatus(ctx context.Context, r *runtime.ContainerSt
 // ContainerStats returns stats of the container. If the container does not
 // exist, the call returns an error.
 func (c *CriManager) ContainerStats(ctx context.Context, r *runtime.ContainerStatsRequest) (*runtime.ContainerStatsResponse, error) {
-	return nil, fmt.Errorf("ContainerStats Not Implemented Yet")
+	containerID := r.GetContainerId()
+
+	container, err := c.ContainerMgr.Get(ctx, containerID)
```

Comment:
Could we use a function to get the `cs`? If we have the function, we will reuse the function in the `ListContainerStats`. Don't repeat yourself.

Reply:
Maybe it is not necessary to do that, the function `getContainerMetrics` has been encapsulated, what has been repeated is just error handling.

Reply:
I have to admit it's more elegant. Thank you.